README.md

```diff
@@ -43,6 +43,11 @@ Displays the LLDP info for each vmnic of VMHosts provided
 **Get-VMCpuReadyPercent**  
 Calculates the CPU ready percent average of virtual machines provided
 
+**Get-VMCpuReadyPercent**  
+Calculates the CPU ready percent average of virtual machines provided
+
+**Get-AllVMsnapshot**  
```

Comment:
Please rename the function to Get-SnapshotAll. This helps with sorting, the included command is Get-Snapshot.

Reply:
Thinking Get-SnapshotHidden might be better.